scriptmodules/libretrocores/lr-mame2003-plus.sh

```diff
@@ -12,21 +12,61 @@
 rp_module_id="lr-mame2003-plus"
 rp_module_desc="Arcade emu - updated MAME 0.78 port for libretro with added game support"
 rp_module_help="ROM Extension: .zip\n\nCopy your MAME roms to either $romdir/mame-libretro or\n$romdir/arcade"
-rp_module_licence="NONCOM https://raw.githubusercontent.com/libretro-mirrors/mame2003-plus-libretro/master/docs/mame.txt"
-rp_module_section="exp"
+rp_module_licence="NONCOM https://github.com/arcadez/-mame2003-plus-libretro/blob/master/docs/mame.txt"
+rp_module_section="main"
 
 function sources_lr-mame2003-plus() {
-    gitPullOrClone "$md_build" https://github.com/libretro-mirrors/mame2003-plus-libretro.git
+    gitPullOrClone "$md_build" https://github.com/arcadez/-mame2003-plus-libretro
 }
 
 function build_lr-mame2003-plus() {
-    build_lr-mame2003
+    rpSwap on 750
+    make clean
+    local params=()
+    isPlatform "arm" && params+=("ARM=1")
+    make ARCH="$CFLAGS" "${params[@]}"
+    rpSwap off
+    md_ret_require="$md_build/mame2003-plus_libretro.so"
```

Comment:
this should still use the parent module function with some changes to avoid duplication (as with the configure).

I will do the code changes to the module as it will be quicker than going to and fro with the PR.

Reply:
Ok and thanks